### PR TITLE
[ZM/Quest] Divine Might Bugfix + Small Adjustments

### PR DESCRIPTION
**Pull request by The-Aerec**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?
Fixes a BCNM startup issue with the Divine Might BCNM. (Starting event on bcnm start that ends bcnm)
Takes control of AA GK during Meikyo to attempt to perform a Light SC on Players
Fixes positions of mobs based on capture data for arena 1. (Adjusted like positions for Arena 2/3 based on position differences in arena 1)

Capture in question from FFXICaptures discord:
https://drive.google.com/file/d/1R9-99fUf_ssEJYlnFyTlyFLbt5QaVR7F/view

## Steps to test these changes
Divine might or AA GK fight and watch him 2hr~
Divine might BCNM for new positions and successful BCNM

### CODE REVIEW

**Review by claywar — APPROVED** [review]

File: scripts/zones/LaLoff_Amphitheater/mobs/Ark_Angel_GK.lua [hunk 1]
```diff
@@ -30,8 +31,24 @@ entity.onMobEngaged = function(mob, target)
     end
 end
 
+
+
 entity.onMobFight = function(mob, target)
-    -- TODO: AA GK actively seeks to skillchain to Light off of his own WSs under MS, or other AA's WSs.
+    if mob:hasStatusEffect(xi.effect.MEIKYO_SHISUI) then
+        if mob:getLocalVar("order") == 0 then
+            mob:useMobAbility(946)
```
Comment (TeoTwawki) [inline scripts/zones/LaLoff_Amphitheater/mobs/Ark_Angel_GK.lua]: `mob:useMobAbility(946) -- Tachi: Yukikaze`

File: scripts/zones/LaLoff_Amphitheater/mobs/Ark_Angel_GK.lua [hunk 2]
```diff
@@ -30,8 +31,24 @@ entity.onMobEngaged = function(mob, target)
...
+            mob:setLocalVar("order", 1)
+            mob:setTP(2000)
+        elseif mob:getLocalVar("order") == 1 then
+            mob:useMobAbility(947)
```
Comment (TeoTwawki) [inline scripts/zones/LaLoff_Amphitheater/mobs/Ark_Angel_GK.lua]: `mob:useMobAbility(947) -- Tachi: Gekko`

File: scripts/zones/LaLoff_Amphitheater/mobs/Ark_Angel_GK.lua [hunk 3]
```diff
@@ -30,8 +31,24 @@ entity.onMobEngaged = function(mob, target)
...
+            mob:setLocalVar("order", 2)
+            mob:setTP(1000)
+        elseif mob:getLocalVar("order") == 2 then
+            mob:useMobAbility(948)
```
Comment (TeoTwawki) [inline scripts/zones/LaLoff_Amphitheater/mobs/Ark_Angel_GK.lua]: `mob:useMobAbility(948) -- Tachi: Kasha`

**Comment by TeoTwawki** [comment]
gj, ty, and I can't wait to taste the players delicious tears.

**Review by TeoTwawki — APPROVED** [review]

**Pull request merged**